### ISSUE TITLE
Displayed math becomes inline in docx

### ISSUE THREAD

**Issue by masuday** [lierdakil/pandoc-crossref#192]
A displayed math becomes inline with an equation number when converting Markdown to docx. No problem in the conversion to LaTeX. Is there any workaround?

![sample](https://user-images.githubusercontent.com/6868817/43853994-a6837c60-9b0f-11e8-9e3c-fb1baf5b903e.png)

Source:
```
The following is a displayed equation.
$$ \mathbf{y}=\mathbf{Xb}+\mathbf{e} $$
When I put an equation number
$$ \mathbf{y}=\mathbf{Xb}+\mathbf{e} $$ {#eq:ref}
it becomes in-line in docx.
```

Command line
```
pandoc -F pandoc-crossref sample.md -o sample.docx
```

My environment:
```
Windows 7
pandoc.exe 2.2.1
Compiled with pandoc-types 1.17.4.2, texmath 0.11, skylighting 0.7.0.2
pandoc-crossref v0.3.2.1 built with Pandoc v2.2.1, pandoc-types v1.17.5 and GHC 8.4.3
```
I have the same result with pandoc-crossref v0.4.0.0-alpha2.

**Comment by pstjohn** [comment]
Can confirm I'm also seeing this. On a mac, `pandoc-crossref v0.3.2.1 built with Pandoc v2.2.1, pandoc-types v1.17.5.1 and GHC 8.4.3`

**Comment by lierdakil** [comment]
So, docx display math is a bit weird compared to LaTeX. It has to be in a paragraph of its own. While Pandoc tries to do its best to split paragraphs containing math, apparently an extra span confuses it. I've created an issue upstream. Until that's fixed, manually putting display math into a separate paragraph should work I think (can't test, no Word at hand right now). Try something like this:

```markdown
The following is a displayed equation.
$$ \mathbf{y}=\mathbf{Xb}+\mathbf{e} $$
When I put an equation number

$$ \mathbf{y}=\mathbf{Xb}+\mathbf{e} $$ {#eq:ref}

it becomes in-line in docx.
```

Another option is to use `tableEqns` option, but it's a wee bit wonky with docx, so YMMV.

**Comment by masuday** [comment]
Thank you for your quick reply. Yes, this workaround works. I am happy to wait for a fix.

**Comment by masuday** [comment]
I tried new pandoc (2.3) with pandoc-crossref (0.4.0.0-alpha2) in a small file and it successfully worked as I expected! I will try bigger files.
Thank you for your escalating this issue to the upstream.

**Comment by lierdakil** [comment]
Cool, thanks for letting me know. One caveat though: for pandoc-2.3, the only release known to work is freshly released v0.3.3.0. All other currently released versions, including v0.4.0.0-alpha2, can (and probably will) fail subtly and silently.

**Comment by lierdakil** [comment]
@tolot27, for instance, `\listoffigures` (or any other list-of commands)
